app/controllers/robot_directory_app.rb

```diff
@@ -17,8 +28,55 @@ class RobotDirectoryApp < Sinatra::Base
     erb :index
   end
 
+  post '/robots' do
+    robot_directory.create(params[:robot])
+    @robot = robot_directory.find(robot_directory.all.last.serial_number)
+    unless params[:email].empty?
+      Pony.mail({:to => params[:email],
```

Comment:
1. this is AWESOME 🎉 
2. this probably should live somewhere other than the controller - you could have a module that takes care of it or even a Plain Old Ruby Object (PORO) - so a class that doesn't map to a table in your database, but that just handles the logic of sending an email